RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernels.cu

```diff
@@ -0,0 +1,309 @@
+#include "RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernelsImpl.h"
+
+template <>
+void CAHitNtupletGeneratorKernelsGPU::fillHitDetIndices(HitsView const *hv, TkSoA *tracks_d, cudaStream_t cudaStream) {
+  auto blockSize = 128;
```

Comment:
I know it probably does not matter, but couldn't we define a `constexpr` for this pretty stable number ?

Reply:
I don't think it makes any differences for a local variable - the compiler knows anyway that it's not being modified.

We will need to review all block sizes (and potentially make them configurable and/or auto-optised base don some heuristic), so I'd rather not make more changes now.